front/src/components/Task/TaskTable.vue

```diff
@@ -108,4 +108,8 @@ export default defineComponent({
 });
 </script>
 
-<style scoped></style>
+<style scoped>
+.q-table td, .q-table th {
+      white-space: normal !important;
```

Comment:
Not a big fan of `!important` in CSS.
Do you think we can solve the problem in a different way ?

Reply:
The `!important` was not required so I have removed it :)